crates/bevy_winit/src/lib.rs

```diff
@@ -288,6 +325,11 @@ pub fn winit_runner_with(mut app: App) {
                     return;
                 };
 
+                world
+                    .get_resource_mut::<WinitPersistentState>()
+                    .unwrap()
```

Comment:
Did you miss this sneaky one `unwrap()`? 🙂

Reply:
Unfortunately `world.cell()` does not implement the infallible `.resource_mut()`, although I should change that to an `expect`. Thanks! 😄

Reply:
This section is filled with pre-existing `unwrap`s. It might be better to implement the infallible functions on `world.cell()` instead of filling this section with repeated `.expect("unable to access resource")`.

Reply:
@Weasy666 I've created #4104 to address this. Good spot!